tests/test_one_lightcurve.sh

```diff
@@ -0,0 +1,17 @@
+# A general test example script.
+# Update the variables in the script to match your local system locations for DATADIR.
+# In DATADIR should be the subdirectories 'images' and 'truth'.
+DATADIR=${HOME}/data/RomanDESCSims
+
+# In INFODIR should be transient_info_table.csv
+#                      transient_host_info_table.csv
+# We have these in tests/data for a few transients, so we can use those to test
+INFODIR=tests/data/RomanDESC
+DATASET=RomanDESC
+oid=41024123441
+# Run from base directoroy
+python python/RomanDESCForwardModelLightcurves.py \
```

Comment:
This should go up a directory first:
```
python ../python/RomanDESCForwardModelLightcurves.py \
```

Also, typo in "directory" in the comment.

Reply:
> This should go up a directory first:

That's what "run from base directory" was meant to say.

The idea is that you would run
```
sh tests/test_one_lightcurve.sh
```
like it says in the README.md

This has come up a couple of times, but I'm not sure how to clearly say the expectation that everything is run from the base directory ('sn-forward-model') not the subdirectories ('python', 'tests', etc.).  Suggestions?